server/build.gradle

```diff
@@ -101,6 +101,7 @@ dependencies {
         exclude(module: 'httpclient')
         //excluded to avoid bringing in jackson core < 2.9
         exclude(module: 'predix-event-hub-sdk')
+        exclude(group: 'org.hibernate')
```

Comment:
after this exclusion, what version of hibernate validation library will be available on classpath?